terraform/050-pw-manager/outputs.tf

```diff
@@ -1,14 +1,3 @@
-output "ui_bucket" {
-  value = aws_s3_bucket.ui.arn
-}
-
-output "cloudfront_distribution_id" {
-  value = aws_cloudfront_distribution.ui[0].id
-}
-
-output "ui_hostname" {
```

Comment:
This is needed by 060-simplsamlphp. Do you have a plan for replacing it?

Reply:
Yes, it will come from the cloudflare-pages module, which is outside this module.